canonical-web-and-design/snapcraft.io
# Replace password toggle component

## Done
- Removed PasswordToggle component in favour of the React components library one
- Drive by: Fixed a warning about an unsupported CSS property value when building the project

## QA
- Go to https://snapcraft-io-3895.demos.haus/admin/ahnuP3quahti9vis8aiw/settings
- Check that there is a password field for the store ID which is read only

## Issue
Fixes #3716
